src/UIKit/UIGestureRecognizer.cs

```diff
@@ -168,7 +168,11 @@ public void RemoveTarget (Token token)
 		//
 		public IEnumerable<Token> GetTargets ()
 		{
-			return (IEnumerable<Token>) recognizers?.Keys ?? Array.Empty<Token> ();
+			if (recognizers?.Keys is null) {
+				return Array.Empty<Token> ();
+			} else {
+				return (IEnumerable<Token>) recognizers.Keys;
+			}
```

Comment:
With your change we'll fetch the  `Keys` property twice. This is somewhat unnecessary, so changing the code to this will avoid the double work:

```suggestion
			var keys = recognizers?.Keys;
			if (keys is null)
				return Array.Empty<Token> ();
			return (IEnumerable<Token>) keys;
```